package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "sanctuary-def",
+  "version": "0.0.0",
+  "description": "Define strict, curried functions",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/plaid/sanctuary-def.git"
+  },
+  "dependencies": {
+    "ramda": "0.18.x"
```

Comment:
I did a very rough calculation and sanctuary-def only uses about 20 per cent of the functions in Ramda, is the full dependency warranted? I know you don't like build processes but I'd like sanctuary-def to be as thin as possible (I have a project I contribute to in mind but I don't think the owner would consent to a large dependency like Ramda) so it would be nice to have an option of an ultraminimal version.